apps/marketplace/pages/BriefResponsePage.js

```diff
@@ -248,6 +248,7 @@ class BriefResponsePage extends Component {
                       this.props.changeModel(`${this.props.model}.${field}`, `${withGst}`)
                     }}
                     fileCount={this.state.otherDocumentFileCount}
+                    buttonAddedCount={this.state.otherDocumentFileCount}
```

Comment:
This file shouldn't be changing as part of this work.  Can you please revert these changes?

Reply:
reverted to master codes